pkg/ggql/root.go

```diff
@@ -143,11 +162,11 @@ func (root *Root) getReflectType(meta reflect.Type) (obj Type) {
 // capitalizing the GraphQL field name. It can also be used to change the
 // order of the arguments to the field.
 func (root *Root) RegisterField(gqlType, gqlField, goField string, args ...string) (err error) {
-	obj, err := root.getObjType(gqlType)
+	obj, _, err := root.getObjType(gqlType) // TBD input also
```

Comment:
Should an issue be created for this TBD?

Reply:
or address it, yes, thanks

Reply:
Not needed. There is no reason to register fields in an input type. Comment removed.